canonical/canonical.com
# Build /solutions/ubuntu-os and modal form

## Done

- Built /ubutnu-os page and respective modal

## QA

- View the site locally in your web browser at: https://canonical-com-1314.demos.haus/solutions/ubuntu-os
- Compare against [doc](https://docs.google.com/document/d/1X3osSqcm3rWLqYUJr5G4fXEEHCby-j5FEJwClLNx5S4/edit#heading=h.e9t2yuuj3uc1) and [design](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/24.10-Canonical.com%2Fsolutions?node-id=123-2421&m=dev)
- Click on the main CTA's and check form against [doc](https://docs.google.com/document/d/1ne8XUT9GMtMEJZSpv-Uram1-YfOM-GoESeUgvPcG2XQ/edit)
- See that submission goes through as expected

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12723
